docs/configuring.md

```diff
@@ -0,0 +1,81 @@
+# Configuration
+
+RedisAI supports both run-time configuration options and others that should be specified when loading the module. 
+
+## Configuration Options Only Available During Loading
+
+In general, passing configuration options is done by appending arguments after the `--loadmodule` argument in the command line, `loadmodule` configuration directive in a Redis config file, or the `MODULE LOAD` command. For example:
+
+In redis.conf:
+
+```
+loadmodule redisai.so OPT1 OPT2
+```
+
+From redis-cli:
+
+```
+127.0.0.6379> MODULE load redisai.so OPT1 OPT2
+```
+
+From command line:
+
+```
+$ redis-server --loadmodule ./redisai.so OPT1 OPT2
+```
+
+
+## THREADS_PER_QUEUE {number}
+
+Enable configuring the main thread to create a fixed number of worker threads up front per device. This controls the maximum number of threads to be used for parallel execution of independent different operations. 
+
+
+This option can significantly improve the model run performance for simple models.
```

Comment:
Maybe it would be useful to explain why we anticipate gains for _simple_ models (and what we mean by simple)

Reply:
fixed on the latest commit @lantiga , please revise